packages/themes/examples/preview/index.js

```diff
@@ -23,7 +23,17 @@ const colorNameLookup = Object.keys(colors).reduce(
   }),
   {}
 );
-const tokens = Object.keys(themes[Object.keys(themes)[0]]);
+const allTokens = themes[Object.keys(themes)[0]];
+const filteredTokens = {};
+
+// Filter out non-color variables:
+Object.entries(allTokens).forEach(([key, value]) => {
+  if (typeof value === 'string' && value.startsWith('#')) {
+    filteredTokens[key] = value;
+  }
+});
+
+const tokens = Object.keys(filteredTokens);
```

Comment:
```suggestion
const tokens = Object.keys(allTokens).filter((key) => {
  const value = allTokens[key];
  if (typeof value === 'string' && value.startsWith('#')) {
    return true;
  }
  return false;
});
```

Let me know if this refactor doesn't make sense! 

Similarly, we may be able to use `unstable__meta` (which should be `unstable_meta` lol) to get the color token names instead of detecting them, I believe we use this for test verification so should be stable 👀 

```js
import { unstable__meta as meta } from '@carbon/themes';

const colorTokens = meta.colors.flatMap(color => color.tokens);
```